### PR TITLE
refactor(ovhPabx) Update popover & line-config w/ common-style

### PR DESCRIPTION
**Pull request by GenghisJean**

### CODE REVIEW

File: client/app/common/less/popover.less [hunk 1]
```diff
@@ -81,6 +81,12 @@
                         color: #999;
                     }
 
+                    .large-popover-item {
```
Comment (antleblanc) [inline client/app/common/less/popover.less]: Please use helper class instead: `class="border-left-0 border-right-0 w-100"`

File: client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference-configuration.html [hunk 2]
```diff
@@ -15,7 +15,7 @@ <h2 data-translate="telephony_number_feature_conference_popover_title"></h2>
                 <div class="section-group">
                     <!-- LANGUAGE -->
                     <div class="section-item">
-                        <div class="form-group no-space">
+                        <div class="form-group p-0 m-0">
```
Comment (antleblanc) [inline client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference-configuration.html]: Using `p-0` with `form-group` make sense? I think you can just stick with `m-0`.

File: client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference-configuration.html [hunk 3]
```diff
@@ -199,18 +199,19 @@ <h2 data-translate="telephony_number_feature_conference_popover_title"></h2>
...
-                        data-translate="submit">
-                </button>
-
-                <button type="button"
-                        class="btn btn-default"
-                        data-translate="cancel"
-                        data-ng-click="$ctrl.onPopoverCancel()">
-                </button>
+            <footer class="clearfix">
+                <div class="btn-group pull-right">
```
Comment (antleblanc) [inline client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference-configuration.html]: `pull-right` => `float-right`

File: client/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/edit/telephony-group-number-feature-ovh-pabx-dialplan-edit.html [hunk 4]
```diff
@@ -18,132 +18,120 @@ <h2 data-translate="telephony_number_feature_ovh_pabx_dialplan_edit"></h2>
...
-                        data-translate="{{ $ctrl.dialplan.status === 'DRAFT' ? 'common_create' : 'modify' }}">
-                </button>
-
-                <button type="button"
-                        class="btn btn-default"
-                        data-translate="cancel"
-                        data-ng-click="$ctrl.onCancelBtnClick()">
-                </button>
+            <footer class="clearfix">
+                <div class="btn-group pull-right">
```
Comment (antleblanc) [inline client/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/edit/telephony-group-number-feature-ovh-pabx-dialplan-edit.html]: `pull-right` => `float-right`

File: client/app/common/less/popover.less (hunk 1 above)
Comment (GenghisJean) [reply client/app/common/less/popover.less]: this is the equivalent of inline css, I really think it's not in our interest

File: client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference-configuration.html (hunk 2 above)
Comment (GenghisJean) [reply client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference-configuration.html]: @feoche

Comment (feoche) [reply client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference-configuration.html]: Margin or padding should not be necessary here, right? `form-group` has already its own definition of it.

File: client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference-configuration.html (hunk 3 above)
Comment (GenghisJean) [reply client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference-configuration.html]: This is exactly the same thing, https://bootstrapdocs.com/v3.3.5/docs/css/#helper-classes-floats
Also, pull-right is use 61 other times in 57 files and this is without components

File: client/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/edit/telephony-group-number-feature-ovh-pabx-dialplan-edit.html (hunk 4 above)
Comment (GenghisJean) [reply client/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/edit/telephony-group-number-feature-ovh-pabx-dialplan-edit.html]: This is exactly the same thing, https://bootstrapdocs.com/v3.3.5/docs/css/#helper-classes-floats
Also, pull-right is use 61 other times in 57 files and this is without components

**Pull request merged**